fetch.bs

```diff
@@ -1162,6 +1162,18 @@ Unless stated otherwise, it is "<code>basic</code>".
 <p>A <a for=/>request</a> has an associated <dfn export for=request id=done-flag>done flag</dfn>.
 Unless stated otherwise, it is unset.
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=reload-navigation-flag>reload-navigation flag</dfn>. Unless stated
+otherwise, it is unset.
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=history-navigation-flag>history-navigation flag</dfn>. Unless stated
+otherwise, it is unset.
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=submission-navigation-flag>submission-navigation flag</dfn>. Unless stated
+otherwise, it is unset.
```

Comment:
It would be good to add a note here why we need these. I also wonder if they should be tri-state since they don't make sense for the Cache API.

Reply:
Tri-state so it can have an 'unset' value? Seems like it'd be less confusing to just store these values in the cache API, even though it isn't particularly useful.

Reply:
Yeah, to indicate it's not relevant/known. I guess I'm fine with storing more bits though.

Reply:
In general I think any field that is added to `Request` must be stored in cache API.  At least that is what we currently do and I think it would be unexpected for `cache.keys()` to return something that differs from what was passed to `cache.put()`.